src/frontend/apps/impress/src/components/ProConnectButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components';
```

Comment:
I copied the one from visio, because as it is already online, it has therefore been approved. This is a component that I should put in the "ui-kit" package